util/check_tool_requirements.py

```diff
@@ -32,8 +56,39 @@ def get_verilator_version():
         log.error(e.stdout)
         return None
 
-def check_version(tool_name, required_version, actual_version):
-    if required_version is None or actual_version is None:
+
+def pip3_get_version(tool):
+    '''Run pip3 to find the version of an installed module'''
+    cmd = ['pip3', 'show', tool]
+    proc = subprocess.run(cmd,
+                          stderr=subprocess.DEVNULL,
```

Comment:
Remove this line to capture stderr with the option to print it if processing fails.

Reply:
Ok.